docs/reference/search/request/docvalue-fields.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[search-request-docvalue-fields]]
+=== Doc value Fields
+
+Allows to return the <<docvalue,doc value>> representation of a field for each hit, for
+example:
+
+[source,js]
+--------------------------------------------------
+GET /_search
```

Comment:
It'd be *awesome* to have a complete example here that indexes a document, does this search, and then shows what the output looks like. I don't think most users understand what they are getting with this option.